### PR TITLE
Ant-Man Pack Commits

### PR DESCRIPTION
**Pull request by wargfn**
Updated Ant-Man Pack for all Ant-Man Cards. Might need to double check Quantities, not sure if those came over here.

### CODE REVIEW

File: pack/wsp.json [hunk 1]
```diff
@@ -1,219 +1,333 @@
...
+		"card_set_type_name_code": "hero",
 		"position": 6,
-		"quantity": 2,
-		"set_code": "wsp",
 		"set_position": 9,
+		"code": "13006",
+		"name": "Wasp Sting",
+		"real_name": "Wasp Sting",
+		"cost": 2,
 		"text": "<b>Hero Action</b> (attack): If you are in <b>Giant</> hero form, deal a total of 4 damage divided among enemies you choose.<br>\n<b>Hero Action</b> (attack): If you are in <b>Tiny</b> hero form, deal 5 damage to an enemy.",
```
Comment (rdonnelly) [inline pack/wsp.json:187]:
```suggestion
		"text": "<b>Hero Action</b> (attack): If you are in <b>Giant</b> hero form, deal a total of 4 damage divided among enemies you choose.<br>\n<b>Hero Action</b> (attack): If you are in <b>Tiny</b> hero form, deal 5 damage to an enemy.",
```

**Review by rdonnelly — CHANGES_REQUESTED** [review]
Hi! This is great! I've got a few comments:

1. Going off of [CardGameDB](http://www.cardgamedb.com/index.php/marvelchampions/marvel-champions-deckbuilder), it looks like Giant Stomp is actually a 2-of.
1. Whenever `card_set_code` is used, I think it should actually be `set_code`.
1. You will want to run the formatter to clean up the white space and order the keys. Information to run that is in the README.
1. Unless you made any modifications to the TRORS encounter cards, you can remove the `trors_encounters.json` file. It was renamed to `trors_encounter.json` (no 's' on the end) to match the other files and play nice with the formatter.
1. I believe the uses of the `<em>` tag should actually be `<i>` (in `text` fields). In HTML, that likely won't matter, but that would make it more consistent with the other card data entered.
1. Hive Mind (previously entered) is missing some card text: `Play only if you are in <b>Tiny</b> hero form.\n<b>Hero Action</b> <i>(thwart)</i>: Remove 2 threat from a scheme. Remove 1 additional threat from that scheme for each Army of Ants support you control.`

I didn't dig _too_ deep, but everything else is looking pretty decent. 👏

**Comment by Kamalisk** [comment]
Will take a look at this tonight, and hopefully make Ant-Man available.
Thanks for the help.

**Comment by wargfn** [comment]
I have a couple of suggested updates to make form Ryan. I am going to fold
those in shortly and should be ready around 3PM EST.

**Comment by wargfn** [comment]
Okay updates from Ryan just committed, Ant-Man packs should be good to go
now.  Good luck sir.

**Pull request merged**